src/plsql/flow_call_activities.pkb

```diff
@@ -122,13 +114,6 @@ as
        for update of prdg.prdg_diagram_level wait 2
     ;
 
-    /*l_called_dgrm_id := flow_diagram.get_current_diagram 
```

Comment:
intentionally commented out?

Reply:
Yes.  Earlier on in the development we switched from evaluating the called diagram when the callActivity was current object TO evaluating which diagrams would get called for the whole process at process creation.  This populates the flow_instance_diagrams table at process creation.   So no need to evaluate mid-process execution.  The changes on this branch merely took the old code, which was commented out, and deleted it.